k8s/nginx/staging/docs-vs.yaml

```
@@ -1,10 +1,29 @@
+  name: internal-docs
+  namespace: staging
+spec:
+  host: staging-docs.airqo.net
+  upstreams:
+    - name: internal-docs
+      service: airqo-stage-internal-docs-svc
+      port: 80
+  subroutes:
+    - path: /internal-docs
+      action:
+        pass: internal-docs      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)